src/Bundle/CoverallsBundle/Api/Jobs.php

```diff
@@ -132,7 +132,15 @@ public function send()
 
         $jsonPath = $this->config->getJsonPath();
 
-        return $this->upload(static::URL, $jsonPath, static::FILENAME);
+        $url = static::URL;
+
+        $endpoint = getenv('COVERALLS_ENDPOINT');
```

Comment:
When this `COVERALLS_ENDPOINT` env variable is not existed, we have to add the condition about handling work.